pep-0008.txt

```diff
@@ -690,16 +690,17 @@ Comments that contradict the code are worse than no comments.  Always
 make a priority of keeping the comments up-to-date when the code
 changes!
 
-Comments should be complete sentences.  If a comment is a phrase or
-sentence, its first word should be capitalized, unless it is an
-identifier that begins with a lower case letter (never alter the case
-of identifiers!).
+Comments should be complete sentences (unless short).  If a comment is
```

Comment:
I disagree with this. I don't see any justification for allowing this.

(Or should I say "Disagree. No Justification."? ;-))

Reply:
I agree with Paul. I don't think there's ever a reasonable to reason to justify not writing complete sentences, no matter how short the comment is.

Reply:
That means all comments should terminate with a period. Otherwise it is not a complete sentence.

Reply:
@toonarmycaptain yep.

Reply:
I edited to remove reference to short comments not needing a period.

Reply:
I agree @pfmoore. I only specified that to comply with the present iteration which states that "If a comment is short, the period at the end can be omitted."